components/icon/stories/icon.stories.js

```diff
@@ -98,6 +88,34 @@ export default {
 export const Default = IconGroup.bind({});
 Default.args = {};
 
+/**
+ * All icons in the Workflow icon set.
+ */
+export const Workflow = FullIconSetTemplate.bind({});
```

Comment:
What would it take to embed this view on a documentation page in the Guides section of our Storybook? Design tokens and then maybe a new doc for Iconography? We can add to it later with more details on best practices and philosophies as well as spritesheet vs. SVG and a statement on our policy for open source icons vs. ones available only for Adobe employees via A4U.

Reply:
Also does _not_ need to be in this pull request.

Reply:
I'll create a Jira issue for this guides page documentation idea.